### PR TITLE
Fix forging details - Closes #3411, #3357

### LINKED ISSUE #3411: Fix forging details in the delegates page
### Expected behavior
The forging details in the delegates page should display the realtime details about the current round, next forgers, forging stats, details about delegates.

### Actual behavior
The various details shown on the page are dependant on complicated calculations and have been made with some assumptions. Those assumptions are wrong/not valid anymore and we need to refactor those calculations/introduce a better way to calculate those values. 

### Which version(s) does this affect? (Environment, OS, etc...)
Lisk Desktop v2.0.0-beta.1

### PR DESCRIPTION
**Pull request by isalga**
### What was the problem?
This PR resolves #3411

### How was it solved?
- Use timestamp to calculate forging times
- Redesign Round status column
- Update forging times calculation

### How was it tested?
Manually

### CODE REVIEW

**Review by sridharmeganathan — CHANGES_REQUESTED** [review]
@isalga Forging status and forging time is incorrect. Please fix 

image

File: src/utils/api/transaction/btc.js [hunk 1]
```diff
@@ -281,7 +281,7 @@ export const create = ({
     }, 0);
 
     if (unspentTxOutsTotal < estimatedTotal) {
-      reject(new Error('Insufficient (estimated) balance'));
+      return reject(new Error('Insufficient (estimated) balance'));
```
[the remark this answers is not in the document]
Comment (isalga) [reply src/utils/api/transaction/btc.js:284]: DeepScan throws error if we don't

[the remark this answers is not in the document]
Comment (UsamaHameed) [reply src/utils/api/transaction/btc.js:284]: Please read https://deepscan.io/docs/rules/promise-reject-fall-through and only apply `return` after the `reject` not before. Also, remove the `return`s that you added for consistency; if you have to disable eslint for those lines, please do so.

**Review by UsamaHameed — APPROVED** [review]

**Review by sridharmeganathan — APPROVED** [review]
Forging times calculation should be fixed in SDK 5 changes

**Pull request merged**